New Terms: C++ `strcpy()` & `strcmp()` (#869)
Codecademy/docs: content/cpp/concepts/strings/strings.md

  - 'paths/computer-science'
---

> **Strings** are objects that represent sequences of characters. In C++, there are two ways to create strings: `string` class or using C-style character strings.

Reviewer: ```suggestion
**Strings** are objects that represent sequences of characters. In C++, the two ways to create strings are with the `string` class or with [C-style](https://www.codecademy.com/resources/docs/c/strings) character strings.
```
